### ISSUE TITLE
[bug] can not get resources

### ISSUE THREAD

**Issue by stopWarByWar** [aptos-labs/explorer#250]
I can not get resources of address "0x05a97986a9d031c4567e15b797be516910cfcb4156312482efc6a19c0a30c948" with error
![image](https://user-images.githubusercontent.com/73871177/198873752-2356351d-1067-4e7c-83a4-2daf6336ebdd.png)

**Comment by web3-fanz** [comment]
you can change to another node RPC which is config more requests handling

**Comment by gregnazario** [comment]
Yes, looks like this page is freezing at the moment, but no longer giving that error https://explorer.aptoslabs.com/account/0x05a97986a9d031c4567e15b797be516910cfcb4156312482efc6a19c0a30c948/resources

**Comment by gregnazario** [comment]
Note: This now hangs (retrieving all the resources), instead of erroring out.

**Comment by zacharydenton** [comment]
The main bottlenecks in this page appear to be react-json-view and emotion (sx={{}}). It's possible to render all 9999 resources on  https://explorer.aptoslabs.com/account/0x05a97986a9d031c4567e15b797be516910cfcb4156312482efc6a19c0a30c948/resources as (`<pre>{JSON.stringify(resource)}</pre>`) in about 300 ms on my machine (whereas the current implementation crashes the browser tab).
